pccommon/pccommon/tracing.py

```diff
@@ -112,8 +112,10 @@ async def _collection_item_from_request(
     try:
         collection_id_match = collection_id_re.match(f"{url}")
         if collection_id_match:
-            collection_id = collection_id_match.group("collection_id")
-            item_id = collection_id_match.group("item_id")
+            collection_id = cast(
+                Optional[str], collection_id_match.group("collection_id")
+            )
+            item_id = cast(Optional[str], collection_id_match.group("item_id"))
```

Comment:
mypy was flagging this inconsistency locally, despite not being changed in this feature.